src/retrieval_service/app/__init__.py

```diff
@@ -11,5 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-from .app import EMBEDDING_MODEL_NAME, init_app, parse_config
```

Comment:
This was removed because linting, but this breaks the process because of current imports in run_app